controllers/tab/settings/appearance/form/PKPAppearanceForm.inc.php

```diff
@@ -95,12 +100,24 @@ function fetch($request) {
 		$templateMgr->assign('uploadImageLinkActions', $uploadImageLinkActions);
 		$templateMgr->assign('uploadCssLinkAction', $uploadCssLinkAction);
 
-		$themePlugins = PluginRegistry::loadCategory('themes');
-		$themePluginOptions = array();
+		$themePlugins = PluginRegistry::getPlugins('themes');
+		if (is_null($themePlugins)) {
+			$themePlugins = PluginRegistry::loadCategory('themes');
+		}
+		$enabledThemes = array();
+		$activeThemeOptions = array();
 		foreach ($themePlugins as $themePlugin) {
-			$themePluginOptions[basename($themePlugin->getPluginPath())] = $themePlugin->getDisplayName();
+			$enabledThemes[basename($themePlugin->getPluginPath())] = $themePlugin->getDisplayName();
+			if ($themePlugin->isActive()) {
+				$activeThemeOptions = $themePlugin->getOptionsConfig();
+				$activeThemeOptionsValues = $themePlugin->getOptionsValues();
+				foreach ($activeThemeOptions as $name => $option) {
+					$activeThemeOptions[$name]['value'] = isset($activeThemeOptionsValues[$name]) ? $activeThemeOptionsValues[$name] : '';
+				}
+			}
 		}
-		$templateMgr->assign('themePluginOptions', $themePluginOptions);
+		$templateMgr->assign('enabledThemes', $enabledThemes);
+		$templateMgr->assign('activeThemeOptions', $activeThemeOptions);
```

Comment:
For multiple assignments, I prefer to use array syntax:

    $templateMgr->assign(array(
        'a' => $a,
    ));